README.md

```diff
@@ -140,6 +140,19 @@ $ clang \
 -o example example.c
 ```
 
+The builds of the toolchain come packaged with two config files, Omax.cfg and OmaxLTO.cfg.
+When used, these config files enable several build optimisation flags. OmaxLTO.cfg enables LTO
```

Comment:
Any recommendation when to use them (LTO and no LTO) together or separately?

Reply:
I've made reference to how LTO can potentially increase link time and memory usage during linking